newspaper/extractors.py

```diff
@@ -863,12 +863,7 @@ def is_boostable(self, node):
         return False
 
     def walk_siblings(self, node):
-        current_sibling = self.parser.previousSibling(node)
-        b = []
-        while current_sibling is not None:
-            b.append(current_sibling)
-            current_sibling = self.parser.previousSibling(current_sibling)
-        return b
+        return self.parser.previousSiblings(node)
```

Comment:
Good optimization 1 liner :) 💯 👍

Reply:
I didn't even realize that this function was just re-using the previousSiblings code, wow maybe we can just replace the function directly in the future.